### PR TITLE
feat: Add new GLA schemas

### PR DESCRIPTION
**Pull request by augustlindemer**
Includes the following schemas:

- GLA gained & lost structure
- GLA existing and proposed uses 
- GLA building details
- GLA communal spaces
- GLA open spaces
- GLA protected spaces

### CODE REVIEW

**Comment by augustlindemer** [comment]
Strange bug: Existing and proposed uses do not allow me to enter any numbers into the GIA entry fields. Can't see any errors with the schema.

File: editor.planx.uk/src/@planx/components/List/schemas/GLA/CommunalSpace.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      type: "question",
+      data: {
+        title: "Is this unit of communal space lost or gained?",
+        fn: "development",
+        options: [
+          { id: "gained", data: { text: "Gained", val: "gained" } },
+          { id: "lost", data: { text: "Lost", val: "lost" } },
+        ],
+      },
```
Comment (jessicamcinchak) [inline editor.planx.uk/src/@planx/components/List/schemas/GLA/CommunalSpace.ts:15]: my first go at summing identical units by development type was only accounting for the previous three options, but I'll update it to work the same when `gained` & `lost` are options too :+1:

File: editor.planx.uk/src/@planx/components/List/schemas/GLA/ExistingAndProposedUses.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          { id: "unknown", data: { text: "Unknown", val: "unknown" } },
+        ],
+      },
+    },
+    {
+      type: "number",
+      data: {
+        title: "What is the existing gross internal floor area?",
+        units: "m²",
+        fn: "area.existing",
```
Comment (jessicamcinchak) [inline editor.planx.uk/src/@planx/components/List/schemas/GLA/ExistingAndProposedUses.ts:32]: I suspect these `.` separated `fn` are the source of your bug! 

For now, I think we need to write all `fn` & `val` _without_ separators - but I'll work on finding a fix to support this / try to wrap my head around why Daf may not have done it from the start ! 

Feel free to note "ideal" `fn` in comment at line ends like this so we don't forget: 
```
fn: "areaExisting", // area.existing
```

File: editor.planx.uk/src/@planx/components/List/schemas/GLA/CommunalSpace.ts (hunk 1 above)
Comment (jessicamcinchak) [reply editor.planx.uk/src/@planx/components/List/schemas/GLA/CommunalSpace.ts:15]: also nit picky thing: you use `loss` in later schemas, but `lost` here as "development" options - are they actually different in legislation or can we standardise?

Comment (augustlindemer) [reply editor.planx.uk/src/@planx/components/List/schemas/GLA/CommunalSpace.ts:15]: Thanks for spotting. loss/gain is what the data hub has in their technical schema across categories (including resi units), so I updated the lost/gained to loss/gain.

**Review by jessicamcinchak — APPROVED** [review]
Looks great !! Thanks for getting all these in :tada:

**Pull request merged**